CHANGELOG.md

```diff
@@ -24,8 +24,21 @@ Version template:
 
 ## [0.4.0] - UNRELEASED
 
+## [0.3.1] - 2020-12-11
+
+### Fixed
+
+* Rename solr projects and embed its dependencies [[#54]]
+* Fix publication to maven central [[#55]]
+
+[#54]: https://github.com/xenit-eu/alfred-telemetry/pull/54
+[#55]: https://github.com/xenit-eu/alfred-telemetry/pull/55
+
+
 ## [0.3.0] - 2020-12-10
```

Comment:
Add a `YANKED`?

Reply:
Hmm, Yanked implies a critical security issue for which a release was removed to me.
This version was just never published

Reply:
I agree!